src/index.html

```diff
@@ -1,17 +1,214 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script type="text/javascript" src="./main.js"></script>
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="info">
+        <legend>Personal information</legend>
+        <div class="form_field">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              minlength="3"
+              maxlength="32"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form_field">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="Name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form_field">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              value="12"
+              min="1"
+              max="100"
+            >
+          </label>
+        </div>
+
+        <div class="form_field">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="date"
+              placeholder="mm/dd/yyyy"
+              required
+            >
+          </label>
+        </div>
+
+
+        <div class="form_field">
```

Comment:
```suggestion
        </div>

        <div class="form_field">
```
there should not be space between two blocks more then 1 line